pkg/cli/login/loginoptions.go

```diff
@@ -133,6 +133,12 @@ func (o *LoginOptions) getClientConfig() (*restclient.Config, error) {
 		clientConfig.CAData = caData
 	}
 
+	// if user has selected option --insecure-skip-tls-verify=true on purpose, the login should not fail due to TLS verification
+	// errors (like TLS Handshake Timeout for example see issue: https://github.com/openshift/oc/issues/496)
+	if o.InsecureTLS {
+		clientConfig.Insecure = true
+	}
+
```

Comment:
I don't think the comment is necessary (and I don't really like issue/BZ/JIRA links in comments).

Also
```golang
  clientConfig.Insecure = o.InsecureTLS
```

Reply:
Sure

Reply:
Hello,

I've reworked the code & comments, squashed the changes in a single commits and made additional investigations to clarify the root cause of the issue (see 2 comments below) and re-ran the tests.

Shall I open a bug in bugzilla to move this forward ?
Thx